package.json

```diff
@@ -4,26 +4,30 @@
   "private": true,
   "scripts": {
     "dev": "next dev --port 3001",
+    "dev:turbo": "next dev --port 3001 --turbo",
     "build": "next build",
     "start": "next start",
     "lint": "next lint"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.1.1",
     "@types/node": "20.3.1",
     "@types/react": "18.2.13",
     "@types/react-dom": "18.2.6",
     "autoprefixer": "10.4.14",
+    "axios": "^1.4.0",
     "daisyui": "^3.1.5",
     "eslint": "8.43.0",
     "eslint-config-next": "13.4.7",
+    "jsonapi-fractal": "^2.3.1",
     "jwt-decode": "^3.1.2",
     "next": "13.4.7",
-    "next-drupal": "^1.6.0",
```

Comment:
It might be helpful to reintroduce next-drupal back when we start using more "Drupal" stuff.

It seems like it's just slowing you down though, so let's remove it.

Reply:
When RSC is supported for `next-drupal`, we can consider it

Reply:
Sounds good.